desktop/src/player.rs

```diff
@@ -224,6 +224,33 @@ impl ActivePlayer {
                     "DejaVu Sans Mono".into(),
                 ],
             );
+            player_lock.set_default_font(
+                DefaultFont::JapaneseGothic,
+                vec![
+                    "ヒラギノ角ゴ Pro W3".into(), // Mac with Japanese environment
+                    "MS UI Gothic".into(),        // Windows
+                    "Noto Sans CJK JP".into(),    // Linux
+                    "Arial Unicode MS".into(),    // Mac fallback
+                ],
+            );
+            player_lock.set_default_font(
+                DefaultFont::JapaneseGothicMono,
+                vec![
+                    "Osaka－等幅".into(),      // Mac with Japanese environment
+                    "MS UI Gothic".into(),     // Windows
+                    "Noto Sans CJK JP".into(), // Linux
```

Comment:
Considering the name, maybe we should try "Noto Sans Mono CJK JP" first?

Reply:
On Linux? I don't have those fonts at all.

Reply:
What do you get when you run https://raw.githubusercontent.com/hogelog/detect-device-font/22a9333a8ea18e558f2cb519b5fa684157dabecf/DeviceFontTest.swf in Flash Player?

Reply:
![image](https://github.com/ruffle-rs/ruffle/assets/296445/1023f959-4e0b-4b07-b6a8-4760e9be8bd5)